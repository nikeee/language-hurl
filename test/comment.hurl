# this is a comment

# these are also comments:

# "asdasd"
# `asdasd`

# csrf_token: xpath "string(//meta[@name='_csrf_token']/@content)"
# GET https://example.org/api/health
