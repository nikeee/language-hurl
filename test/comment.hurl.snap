># this is a comment
#^^^^^^^^^^^^^^^^^^^ source.hurl comment.line.number-sign.hurl
>